bundles/com.zeligsoft.domain.dds4ccm.ui/src/com/zeligsoft/domain/dds4ccm/ui/wizards/DDS4CCMModelWizardPage.java

```diff
@@ -84,10 +84,26 @@ protected String getDefaultCdtProjectName() {
 	public void createControl(Composite parent) {
 		// TODO Auto-generated method stub
 		super.createControl(parent);
-		final Composite composite = (Composite) getControl();
-		// create a Group and two Radio buttons (Button) within it.
-		createModelTypeGroup(composite);
 		
+		if(isAxciomaSupported()){
+			final Composite composite = (Composite) getControl();
+			// create a Group and two Radio buttons (Button) within it.
+			createModelTypeGroup(composite);
+		}
+	}
+	
+	public boolean isAxciomaSupported(){
```

Comment:
Move this method to a separate class, and make it static. That way, other UI elements can take advantage of it, too.